index.html

```diff
@@ -0,0 +1,578 @@
+  on SCHEMA
+```
+
+### Discernability
+
+All directive forms a spec provides must be **discernable**. That is, a given usage of the directive which is valid according to one form **must not** be a valid usage of any other form.
+
+A single, unique, non-optional argument for each form satisfies this requirement.
+
+While specs may indicate multiple directive forms, including multiple definitions of the same directive in a document is impossible. Instead, authors must include a definition which is compatible with all usages of the directive within the document, in addition to being compatible with the specified forms.
+
+It is possible for specs to define forms which **cannot** be conjuncted within a document. This typically happens when an argument with a particular name has multiple incompatible scalar representations. For example:
+
+```graphql
+#::[Example of non-conjunctible forms]
+enum example__Data {}
+
+directive @example(value: String!) on SCHEMA
+directive @example(value: example__Data!) on SCHEMA
+```
+
+`value: example__Data` must be represented as an unquoted enum value, but `value: String` must be represented as a quoted string. There is no way to combine these two forms of `@example` into a single valid GraphQL directive definition. Accordingly, they cannot both occur within the same document. Similar conflicts occur if an argument is named as both an input object and a scalar, for instance. Spec authors should generally strive to avoid these situations by selecting a new name for the new argument and marking the older form with `@deprecated`:
+
+```graphql
+#::[Example of conjunctible forms]
+enum example__Data {}
+
+directive @example(value: String! @deprecated) on SCHEMA
+directive @example(data: example__Data!) on SCHEMA
+```
+
+## Processing Schemas
+
+```mermaid
+#::[Example processing pipeline]
+graph LR
+  schema(["📄  Input Schema"]):::file-->proc("🤖 &nbsp;Processor")
+  proc-->output(["📄  Output Schema"]):::file
+  classDef file fill:none,color:#eee;
+  style proc fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+A common use case is that of a processor which consumes a valid input schema and generates an output schema.
+
+The general guidance for processor behavior is: don't react to what you don't understand.
+
+Specifically, processors:
+  - **should** pass through `@core` directives which reference unknown spec URLs
+  - **should** pass through prefixed directives, types, and other schema elements  
+
+An exception to this is processors which prepare the schema for final public consumption. Such processors **may** choose to eliminate all unknown directives and prefixed types in order to hide schema implementation details within the published schema. This will impair the operation of tooling which relies on these directives—such tools will not be able to run on the output schema, so the benefits and costs of this kind of information hiding should be weighed carefully on a case-by-case basis.
```

Comment:
This paragraph seems to be kind of talking about the same sort of thing as creating an export schema from a core schema.  Can we avoid having this separate concept of "preparing for final public consumption" and just talk about making the export schema?

Reply:
Ah yes, this language predates the explicit concept of an export schema. Will remove / roll into a longer description of how the export schema is constructed.

Reply:
This hasn't been done.